docs/application-server/changelog.md

```diff
@@ -89,6 +89,11 @@ this field was unique this field has been deprecated. It is still present in
 the database as `email_old` so that the new `email` field (containing the old
 `username` as value) can be updated by hand if needed.
 
+
+**Notice:** This change will break the both ACL plugins for MQTT. The queries
```

Comment:
Would you mind to change this to something like:

```
**Note:** If you have integrated your MQTT broker with the ChirpStack PostgreSQL database,
you must update your SQL queries when upgrading to this release as the `username` field in the
database has been renamed to `email`. Please refer to the [MQTT authentication](../project/guides/mqtt-authentication.md)
documentation for more information.
```

I think this makes it a little bit more clear, as not everybody is using the Mosquitto plugins for authentication.

Reply:
Sure, I have updated with the text you provided